.github/workflows/configstore.yml

```diff
@@ -0,0 +1,57 @@
+name: configstore
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tox-env:
+          - py310
+          - py39
+          - py38
+          - py37
+          - coverage
+          - pkg
+    steps:
+      - uses: actions/checkout@v2.3.5
+      - name: Determine Python version
+        id: get-python-version
+        run: |
+          echo ::set-output name=value::\
+          $(jq -r '.["${{ matrix.tox-env }}"]' < .github/workflows/tox-mapping.json)
```

Comment:
I needed this trick to map `py39` (tox env value) to `3.9` (python version used just below)

tox-gh-action is supposed to handle this but I ran unto a few issues.